build.gradle

```diff
@@ -32,32 +33,38 @@ buildscript {
 
 subprojects {
   afterEvaluate { project ->
-    task sourcesJar(type: Jar) {
-      from android.sourceSets.main.java.srcDirs
-      classifier = 'sources'
-    }
 
-    task javadoc(type: Javadoc) {
-      source = android.sourceSets.main.java.srcDirs
-      classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-      classpath += fileTree(dir: "$buildDir/intermediates/exploded-aar/", include: "**/classes.jar")
-      classpath += configurations.compile
-      // Exclude generated files
-      exclude '**/BuildConfig.java'
-      exclude '**/R.java'
-    }
+    // These two projects are JARed and DOCed using their own build.gradle files
+    if (!project.projectDir.absolutePath.contains("backbone") && !project.projectDir.absolutePath.contains("skin")) {
```

Comment:
Why is it bad to ignore RS' top level gradle?  I don't see anything important in it, and if there is, we should move them into backbone's build.gradle or skin's.  Plus, that is just how gradle works, the top-level is only applicable for running the top-level project.